### PR TITLE
Fix: Add 'key' parameter to S3 tfvars variable

### PR DESCRIPTION
**Pull request by Stretch96**
* This is to follow the format of the s3 tfvars module

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -30,6 +30,7 @@ variable "tfvars_s3_tfvars_files" {
   type = map(
     object({
       path = string
+      key  = optional(string, "")
       }
```
Comment (DrizzlyOwl) [inline variables.tf:34]: can you fix this indentation please :)

**Review by DrizzlyOwl — APPROVED** [review]

**Comment by olivia-work** [comment]
why do we need a 'key' parameter to S3 tfvars variable?

**Comment by Stretch96** [comment]
It's to follow the format of the s3 tfvars module :)

File: variables.tf (hunk 1 above)
Comment (Stretch96) [reply variables.tf:34]: That's what `terraform fmt` sets it to for some reason

**Pull request merged**